CWC 2019 Wiki page (#2286)
ppy/osu-wiki: wiki/Tournaments/CWC/2019/en.md

---
tags:
- CWC 2019
- CWC2019
---

> osu!catch World Cup 2019

Reviewer: ```suggestion
# osu!catch World Cup 2019
```

> =============================

Reviewer: delete

> =============================

Reviewer 2: ```suggestion
```

> ![osu!catch World Cup 2019](img/logo.png)

Reviewer: ```suggestion
![CWC 2019 Logo](img/logo.png)
```

> ![osu!catch World Cup 2019](img/logo.png)

Reviewer 2: This image too big, probably better to use the actual banner like on [this article](https://osu.ppy.sh/help/wiki/Tournaments/TWC/2019) for example

> The **osu!catch World Cup 2019** (**_CWC 2019_**) is a country-based osu!catch tournament hosted by the [osu! team](/wiki/People/The_Team). It is the 8th installment of the osu!catch World Cup.

Reviewer: ```suggestion
The **osu!catch World Cup 2019** (**_CWC 2019_**) is a country-based osu!catch tournament hosted by the [osu! team](/wiki/People/The_Team). It is the eight installment of the osu!catch World Cup.
```

## Tournament Schedule

| Event | Timestamp |

> | ---: | :--- |

Reviewer: ```suggestion
| --: | :-- |
```

> | Registration Phase | 2019-05-28/2019-06-11 |

Reviewer 2: ```suggestion
| Registration Phase | 2019-05-29/2019-06-11 |
```

> | Drawings | 2019-06-22 (14:00 UTC) |

Reviewer: ```suggestion
| Live Drawings | 2019-06-22 (14:00 UTC) |
```

| Group Stage | 2019-06-29/2019-06-30 |
| Round of 16 | 2019-07-06/2019-07-07 |
| Quarterfinals | 2019-06-13/2019-06-14 |
| Semifinals | 2019-07-20/2019-07-21 |
| Finals | 2019-07-27/2019-07-28 |
| Grand Finals | 2019-08-03/2019-07-04 |

## Prizes

> In every world cup, people conquering a place on the podium are eligible to receive unique prizes created specifically for World Cup winners. It is possible that these items change with every installment of the World Cups.

Reviewer: Delete

| Placing | Prize(s) |

> | --- | :--- |

Reviewer: ```suggestion
| :-: | :--- |
```

> | ![Gold Crown](/wiki/shared/GCrown.png "1st place") | $150 per team member, profile badge, "osu!catch Champion" user title for one year |

Reviewer: ```suggestion
| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | $150 per team member, unique profile badge, "osu!catch Champion" user title for one year |
```

Apply to all prizes (unique profile badge)

| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | $80 per team member, Profile badge |
| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | $40 per team member, Profile badge |

> ## Organization

Reviewer: ```suggestion
## Organisation
```

The osu!catch World Cup 2019 is run by various community members by distributing the multitude of tasks into various fields of responsibility.

| Position | Members |

> | ------------ | -------------- |

Reviewer: ```suggestion
| :-- | :-- |
```

| Management | ![][flag_AR] [juankristal](https://osu.ppy.sh/users/443656), ![][flag_NZ] [deadbeat](https://osu.ppy.sh/users/128370) |
| Map Selectors | ![][flag_TN] [-Ken](https://osu.ppy.sh/users/4430811), ![][flag_US] [Ascendance](https://osu.ppy.sh/users/2931883), ![][flag_ES] [Deif](https://osu.ppy.sh/users/318565), ![][flag_CN] [F D Flourite](https://osu.ppy.sh/users/2459589) |
| Commentators | _to be decided_ |
| Referees | ![][flag_ES] [Deif](https://osu.ppy.sh/users/318565), ![][flag_DE] [p3n](https://osu.ppy.sh/users/123703), ![][flag_US] [tigereyes144](https://osu.ppy.sh/users/6499811), ![][flag_CL] [WalterToro](https://osu.ppy.sh/users/5281416), ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973) |
| Statisticians | ![][flag_NZ] [deadbeat](https://osu.ppy.sh/users/128370), ![][flag_DE] [Nwolf](https://osu.ppy.sh/users/1910766) |

## Links

- [Discussion thread](https://osu.ppy.sh/community/forums/topics/913952)
- [Livestream](https://www.twitch.tv/osulive)
- [Sign up here!](https://osu.ppy.sh/community/tournaments/20)

> ------------------------------------------------------------------------

Reviewer: ```suggestion
---
```

3 dashes for every line break(s)

## Mappools

_Mappools won't be available until one week before the stage that it is meant to be played_

------------------------------------------------------------------------

## Ruleset

### Tournament Rules

1. The osu!catch World Cup is a country-based team tournament, played on the osu!catch game mode.
   - While this competition is planned as a 3 versus 3 setup, this might change depending on the amount of incoming registrations.

> 

Reviewer 2: Remove all these new lines in those lists

15. If a player disconnects between the beatmaps and the team can not provide an exchange, the match can be delayed 10 minutes at maximum.
16. All players and referees must to be treated with respect. Instructions of the referees and tournament management is to be followed. Decisions labeled as final are not to be objected.
17. Disrupting the match by foul play, picking inappropriate warmup maps (see below), insulting and provoking other players or referees, delaying the match or other deliberate inappropriate misbehavior is strictly prohibited.
18. The multiplayer chatrooms underlie the [osu! community rules](/wiki/Rules). All chat rules apply to the multiplayer chatrooms.
    - Breaking the chat rules results in a silence. Silenced players can not participate at multiplayer matches and must be exchanged for the time being.

19. In Group stage, 'Win by default' will be considered as win by 5:0, +1.0 score difference ratio.
20. Unexpected incidences are handled by the tournament management. Referees may allow higher tolerance depending on the given circumstances. This is up to their discretion.
21. Penalties for violating the tournament rules can be:

>    - Exclusion of specific players for one map

Reviewer 3: ```suggestion
    - Exclusion of specific players for one map
```

need one more space, same thing for the others below

### Stage Instructions

1. In the first stage (Group Stage), the teams will be divided into 8 groups of 4 teams.
   - This may change depending on how many teams are accepted into the competition at the end.

2. All the teams from each group will face each other.
3. Rankings of each group are determined by sorting the results of each team's performance in the following priority:
   - Most matches won.
   - Have higher `{(the number of beatmaps won) - (the number of beatmaps defeated)}`.
   - Most beatmaps won.
   - **Winner of the match played previously between the tied teams.**
   - In the event of a triple tie:
     - Have higher `∑{(total score difference) / (maximum score)}`.
     - Winner of the rematch.

4. The top 2 teams of each group will move on to the Knock-Out Stages.
   -  This may change with the actual Group Stage setup.

5. Following stages are Double Elimination Stages. This means that the winner moves to the next stage and the losing team gets moved to the Loser bracket.
6. Based on [this image](https://puu.sh/bUq5V/f1066103b0.png), the stages are split up into the following:

| Stage | Match ID |

> | -----: | -------- |

Reviewer: ```suggestion
| --: | :-- |
```

| Round of 16 |  A, B, C, D, E, F, G, H |
| Quarterfinals | I, J, K, L & R, S, T, U |
| Semifinals | M, N & V, W, X, Y, Z, AA |

> | Finals - Week 1 | O & AB, AC, AD |

Reviewer 2: ```suggestion
| Finals | O & AB, AC, AD |
```

> | Finals - Week 2 | AE, P, Q |

Reviewer 2: ```suggestion
| Grand Finals | AE, P, Q |
```

2. Each team is free to select up to one warm-up map. Using beatmaps with questionable content is prohibited. All maps must be osu!catch specific maps.
3. Each captain can ban one beatmap to be selected from the pool **at the start of the match**. These beatmaps are not allowed to be picked by any team in the entire match.  
4. **After a team reaches 3 points the opposition can ban another map as a comeback mechanic.**. This means that both teams can potentially ban up to **two maps** during the entirety of the match.
5. Beatmap selection will alternate between each captain selecting a beatmap out of the map pool.
6. Each captain must use "!roll" once in #multiplayer.
   - **The winner of the !roll decides wheter they want to pick and ban first, or pick and ban second.**

7. Captains may pick freely from any bracket.
   - In case of a tie, the tiebreaker map must be played.

8. Results of the Group Stage will be published via a Statistics sheet.

### Mappool Instructions

1. There will be 1 mappool for the Group Stage, 1 mappool for the Round of 16, 1 mappool for the Quarterfinals, 1 mappool for the Semifinals, 1 mappool for the Finals and 1 mappool for the Grand Finals.

> 

Reviewer 3: these empty spaces between the lists should be removed, it is giving the main list items more margin than the sub-lists

2. Each mappool consists of 4 brackets: NoMod, [Hidden](/wiki/Game_Modifiers), [HardRock](/wiki/Game_Modifiers) and [DoubleTime](/wiki/Game_Modifiers).
3. Mappools for Groups Stage, Round of 16 and Quarterfinals consists of 16 maps in total.
4. **Mappools for Semifinals, Finals and Grand Finals consists of 19 maps in total.**
5. Each mappool has one tiebreaker.
6. The NoMod bracket will be played with no mods activated.
7. The Hidden, HardRock and DoubleTime bracket will be played with the respective modes activated.  
   - When playing a map from the HardRock or DoubleTime pool, Hidden can be used optionally by any player.

8. The tiebreaker will be played with no mods selected or with the Hidden mod.
9. There will be 6 maps in the NoMod bracket for all stages.
10. The size of the [Hidden](/wiki/Game_Modifiers), [HardRock](/wiki/Game_Modifiers) and [DoubleTime](/wiki/Game_Modifiers) brackets will be 3 in Groups Stage, Round of 16 and Quarterfinals.

> 11. The size of the [Hidden](/wiki/Game_Modifiers), [HardRock](/wiki/Game_Modifiers) and [DoubleTime](/wiki/Game_Modifiers) brackets will be 3 in Semifinals, Finals and Grand Finals.

Reviewer 2: Uh, 3 and 3? i think it gets bigger here

### Scheduling Instructions

1. Each stage will be held on **a single weekend**.
2. Matches in Group Stage may overlap.
3. All Double Elimination Stages will be held on either Saturday or Sunday, UTC+0.
   - Finals stage may have matches on Friday, depending on the occuring encounters.
4. Scheduling will be handled by the Tournament Management. Schedules will be released on the Sunday before the first matches of the actual stage. Tournament Management will try to create the schedule to respect the participant's time zone.
   - In the stages Quarterfinals and higher: Please inform tournament management before Sunday, if you expect a specific time slot to be unavailable in the following week. Wishes are tried to be followed, alas no promises can be made.

> 5. **Reschedules will only be considered if both teams agree to a time at least 48 hours before the first match in that particular stage.**

Reviewer 2: We are going back to 48h? if not we should use the TWC rules.

```suggestion
5. **Reschedules will only be considered if both teams agree to a time, this needs to be done and notified to the tournament staff before Wednesday at 23:59 UTC+0 in that particular week when your match takes place.**
```
